### ISSUE TITLE
Zenodo fetch scripts are broken with new Zenodo API

### ISSUE THREAD

**Issue by jluethi** [fractal-analytics-platform/fractal-demos#72]
With the new Zenodo update, the fetching of test data in the demo scripts broke. It now produces this error:

```
Traceback (most recent call last):
  File "/Users/joel/mambaforge/envs/fractal-demo-client-1.3.7/bin/zenodo_get", line 8, in <module>
    sys.exit(zenodo_get())
  File "/Users/joel/mambaforge/envs/fractal-demo-client-1.3.7/lib/python3.10/site-packages/zenodo_get/zget.py", line 309, in zenodo_get
    total_size = sum(f['size'] for f in files)
  File "/Users/joel/mambaforge/envs/fractal-demo-client-1.3.7/lib/python3.10/site-packages/zenodo_get/zget.py", line 309, in <genexpr>
    total_size = sum(f['size'] for f in files)
KeyError: 'size'
```

**Comment by tcompa** [comment]
Ref:
* https://github.com/dvolgyes/zenodo_get/issues/19

**Comment by tcompa** [comment]
Fixed with zenodo-get v1.5
